pkg/readiness/config/config.go

```diff
@@ -71,8 +73,9 @@ func GetLogger() *lumberjack.Logger {
 	logger := &lumberjack.Logger{
 		Filename:   readinessProbeLogFilePath(),
 		MaxBackups: readIntOrDefault(readinessProbeLoggerBackups, 5),
-		MaxSize:    readInt(readinessProbeLoggerMaxSize),
+		MaxSize:    readIntOrDefault(readinessProbeLoggerMaxSize, 5),
```

Comment:
prior to that we set 0, which will default to 100mb. That means the readinessProbe might will up to 5x100mb=500mb of logs. Note - those logs are not that useful, while still filling up the disk